plugins/browse-pvc.yaml

```diff
@@ -0,0 +1,32 @@
+# Krew Manifest
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: browse-pvc
+spec:
+  version: "v0.3.0"
+  homepage: https://github.com/clbx/kubectl-browse-pvc
+  shortDescription: "Browse PVCs from the command line"
+  description: "Browse PVCs from the command line"
```

Comment:
You can add a bit more detail here as to how the plugin does this task.

Reply:
Added